addons/mail/models/mail_thread.py

```diff
@@ -1913,7 +1913,9 @@ def message_post(self, body='', subject=None,
                 if False/0, mail.message model will also be set as False
             :param str body: body of the message, usually raw HTML that will
                 be sanitized
-            :param str type: see mail_message.type field
+            :param str subject: subject of the message
```

Comment:
I mainly decided to update it because I needed to have a better view of the parameters but I will remove that from this task. As you say better do it once completely